src/main/java/net/mcreator/blockly/java/blocks/JavaCodeExecuteBlock.java

```diff
@@ -24,15 +24,14 @@
 import net.mcreator.util.XMLUtil;
 import org.w3c.dom.Element;
 
-public class JavaCodeBlock implements IBlockGenerator {
+public class JavaCodeExecuteBlock implements IBlockGenerator {

```

Comment:
Rename to JavaCodeProceduralBlock

Reply:
:heavy_check_mark: Done

Reply:
You need to rename all locations. Tests fail now: https://travis-ci.org/github/MCreator/MCreator/builds/735693515

Use Shift + F6 in IDEA for proper refactoring.

Reply:
Check my recent commits.